components/button-toggle-group/src/vwc-button-toggle-group.ts

```diff
@@ -85,12 +85,22 @@ export class VWCButtonToggleGroup extends LitElement {
 	}
 
 	set values(values: (string | false | null)[]) {
-		this.clearSelection();
+		function isButtonSelectedWithoutValue(button: Element) {
+			return !button.hasAttribute('value') && button.hasAttribute(SELECTED_ATTRIBUTE_NAME);
+		}
+
+		function isButtonWithDefinedValue(button: Element) {
+			return button.hasAttribute('value') && values.includes(button.getAttribute('value'));
+		}
+
 		if (!this.multi) {
 			values = [values[0]];
```

Comment:
This line still don't allow setting `values` property equals to `null` or `[]` both are meaning empty set of values and are valid real cases. How should we clean the values then if we are not allowed to pass null nor empty array?

Reply:
Please see these two tests:
```
it('should return empty values array when set to null', async function() {
			const emptyValues = [];
			actualElement.multi = true;
			actualElement.values = [buttonValues[0], buttonValues[2]];
			await actualElement.updateComplete;
			actualElement.values = null;
			await actualElement.updateComplete;
			expect(JSON.stringify(actualElement.values)).to.equal(JSON.stringify(emptyValues));
			expect(actualElement.selected.length).to.equal(0);
		});

		it('should return empty values array when set to []', async function() {
			const emptyValues = [];
			actualElement.multi = true;
			actualElement.values = [buttonValues[0], buttonValues[2]];
			await actualElement.updateComplete;
			actualElement.values = [];
			expect(JSON.stringify(actualElement.values)).to.equal(JSON.stringify(emptyValues));
			expect(actualElement.selected.length).to.equal(0);
		});
		```
Aren't they testing these exact use cases?

Reply:
cool thank you so much 👍